mikeio/dfs.py

```diff
@@ -0,0 +1,30 @@
+from .helpers import safe_length
+from .dutil import Dataset, get_item_info, get_valid_items_and_timesteps
+from .dotnet import (
+    to_numpy,
+    to_dotnet_float_array,
+    to_dotnet_datetime,
+    from_dotnet_datetime,
+)
+
+class Dfs123:
+
+    _projstr = None
+    _start_time = None
+
+    def __init__(self, filename = None):
+        self._filename = filename
+        # TODO self._read_header(filename)
```

Comment:
Call this?

Reply:
Yes, there is still some cleaning up to do...